conda-recipe/meta.yaml

```diff
@@ -1,7 +1,4 @@
-{% set required_compiler_version = "2024.0" %}
-{% set excluded_compiler_version1 = "2024.0.1" %}
-{% set excluded_compiler_version2 = "2024.0.2" %}
-{% set excluded_compiler_version3 = "2024.0.3" %}
+{% set required_compiler_version = "2024.2.0" %}
```

Comment:
Should `Install example requirements` step in `.github/workflows/conda-package.yml` be updated in the same way to update `dpcpp_linux-64` dependency?

Reply:
Yes, great catch. I am pushing a fix.